mesheryctl/internal/cli/root/system/system.go

```diff
@@ -27,7 +27,9 @@ import (
 )
 
 const (
-	fileURL = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	fileURL         = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
```

Comment:
rename to dockerComposeURL?

Reply:
Yes. This URL will be made dynamic for supporting contexts i.e `channel` and `version`. So, we will need to change this. I will definitely make this better in a next PR where I will also add proper context support.

Reply:
looking forward to that :smile:

Reply:
This will also affect the `system reset` command. Currently, the reset command is "context aware". But since the `system start` was not fully "context aware", this was not handled more "gracefully". But definitely this should be reworked.